### ISSUE TITLE
Animation!! (Bumps & Potholes)

### ISSUE THREAD

**Issue by zipzit** [SebWiers/motorcycle-front-end#1]
What you have already provided is pretty awesome, but a front end calculator really needs animation for bumps and pothole motion.... 

Check out https://dogfeatherdesign.com/engineering-projects/mechanisms-mechanical-walker/ for javascript implementations of mechanisms in motion.   Look at the bottom few iframes.  The javascript is fully commented, so you can figure out how the thing works. e.g. check out https://dogfeatherdesign.com/dfd_mechanisms/strandbeest.html

Nice work in your repo on what you are currently sharing.  Many thanks!

**Comment by SebWiers** [comment]
Yep, that's the eventual plan.   Right now its not doing any sort of kinematics, and doesn't even have the relevant inputs.  I'm not sure such a kinematics tool is really needed for the case of simple slider forks, since the kinematics don't really change with wheel travel (though can change as the entire frame pitches).  However, for designs that use pivots instead of / in combination with sliding elements (springers, girders, Telelever, Duolever, Yamaha GTS, 2018 Goldwing) you get dynamic changes to both geometry and kinematics as the wheel moves through travel.  This is what I'm ultimately aiming to model, in the same vein as the software I used to figure out the design for the Hossack style front end I built on the project seen at https://hackaday.io/project/11706-secapocalypse

Its funny you should link to the dogfeather site, because I literally talked on the phone with the site creator before starting my work regarding design approaches.  I forget how I came across it, but it was one of the things that showed me that yeah, this was a workable project.

I'm starting to think I maybe want to use SVG instead of canvas though, as it may allow for easier manipulation of individual drawing elements.
